### DISCUSSION TITLE
Is there a way to paginate the scheduled jobs?

### DISCUSSION THREAD

**Discussion by italktothewind** [kagkarlsson/db-scheduler#465] [Q&A]
Suppose I have 100.000 jobs. I want to show a list of them in some UI. I cannot ask for 100.000 jobs once, instead I would like to ask for pages of 100 jobs each. Is there any way? Thanks!

**Comment by kagkarlsson** [comment]
No, not currently. Open to contributions, ideally not limit+offset-based.

Currently, I think all items are fetched in https://github.com/bekk/db-scheduler-ui
